config.go

```diff
@@ -279,14 +279,19 @@ func updateConfigFromClowder(configuration *ConfigStruct) {
 		fmt.Println(noKafkaConfig)
 	} else {
 		// make sure broker(s) are configured in Clowder
-		if len(clowder.LoadedConfig.Kafka.Brokers) > 0 {
-			broker := clowder.LoadedConfig.Kafka.Brokers[0]
+		clowderBrokers := len(clowder.LoadedConfig.Kafka.Brokers)
```

Comment:
- "numBrokers" would be a better name.
- we could also use 'clowder.KafkaServers', which is a []string with the broker addresses, directly